modules/nw-kube-proxy-sync.adoc

```diff
@@ -0,0 +1,14 @@
+// Module included in the following assemblies:
+// * networking/configuring-kubeproxy.adoc
+
+[id="modifying-kubeproxy-sync-{context}"]
+= About iptables rules synchronization
+
+The synchronization period determines how frequently the Kubernetes network
+proxy (kube-proxy) syncs the iptables rules on a node.
+
+A sync is triggered when either of the following occurs:
```

Comment:
s/A sync is triggered/A sync begins
^ Use a simple tense. I question if "triggered" will translate well.

s/following/following events